lib/gravity/gravity.go

```diff
@@ -17,70 +17,129 @@ limitations under the License.
+	if err := json.Unmarshal([]byte(infoS), &info); err != nil {
+		return trace.Wrap(err)
+	}
+	if !info.RemoteSupportConfigured {
+		log.Info("Remote support is not configured.")
+		return nil
 	}
-	return trace.Wrap(err)
+	action := "disable"
+	if enabled {
+		action = "enable"
+	}
+	out, err := gravityCommand("tunnel", action)
+	if err != nil {
+		return trace.Wrap(err, "failed to %v tunnel: %s", action, out)
+	}
+	log.Infof("Remote support %vd: %s.", action, out)
+	return nil
+}
+
+// clusterInfo is a subset of the local cluster info
+type clusterInfo struct {
+	// RemoteSupportConfigured indicates whether local cluster has remote
+	// support configured (enabled or disabled trusted cluster)
+	RemoteSupportConfigured bool `json:"remoteSupportConfigured"`
 }
 
 // CompleteInstall marks the site installation step as complete.
 func CompleteInstall(support bool) error {
-	siteName, err := GetLocalSite()
+	// remote support actions are available only in enterprise edition
+	isEnterprise, err := IsEnterprise()
 	if err != nil {
-		return trace.Wrap(err)
+		return trace.Wrap(err, "failed to determine edition")
+	}
+	if isEnterprise {
+		err := ConfigureRemoteSupport(support)
+		if err != nil {
+			return trace.Wrap(err, "failed to configure remote support")
+		}
 	}
-	action := "on"
-	if !support {
-		action = "off"
+	out, err := gravityCommand("site", "complete")
+	if err != nil {
+		return trace.Wrap(err, "failed to complete install: %s", out)
 	}
-	out, err := gravityCommand("site", "complete", siteName, fmt.Sprintf("--support=%v", action))
-	log.Infof("complete install output: %s", string(out))
-	return trace.Wrap(err)
+	log.Infof("Install completed: %s.", out)
+	return nil
 }
 
-// GetSiteInfo returns a JSON-formatted string with the site information.
-func GetSiteInfo() (string, error) {
-	siteName, err := GetLocalSite()
+// GetClusterInfo returns a JSON-formatted string with the local cluster information.
+func GetClusterInfo() (string, error) {
+	out, err := gravityCommand("site", "info", "--output=json")
 	if err != nil {
-		return "", trace.Wrap(err)
+		return "", trace.Wrap(err, "failed to get cluster info: %s", out)
 	}
-	out, err := gravityCommand("site", "info", siteName, "--output=json")
-	log.Infof("get site info output: %s", string(out))
-	return string(out), trace.Wrap(err)
+	log.Infof("Local cluster info: %s.", out)
+	return string(out), nil
 }
 
-// GetLocalSite returns the name of the locally installed site.
-func GetLocalSite() (string, error) {
-	out, err := gravityCommand("local-site")
-	log.Infof("get local site output: %s", string(out))
-	return string(out), trace.Wrap(err)
+// IsEnterprise returns true if the local Telekube cluster is of enterprise
+// edition.
+func IsEnterprise() (bool, error) {
+	out, err := gravityCommand("version")
+	if err != nil {
+		return false, trace.Wrap(err, "failed to determine gravity version: %s", out)
+	}
+	return strings.Contains(strings.ToLower(string(out)), "enterprise"), nil
```

Comment:
Can we make `version` additionally format the version string to avoid ad-hoc parsing/guessing?